lib/src/main/scala/spinal/lib/bus/localbus/sim/MemVIP.scala

```diff
@@ -0,0 +1,108 @@
+package spinal.lib.bus.localbus
+
+import spinal.core._
+import spinal.core.sim._
+class MemVIP{
+  private val mem = scala.collection.mutable.HashMap[Long, BigInt]()
```

Comment:
it would make sense if this would reuse `SparseMemory`

Reply:
As a Mem VIP (Verification IP), it must be Sparse, and here we emphasize that VIP ， so did the MemVIP be better

Reply:
I'm sorry - I don't understand what you are saying.